name,value,relevant log,explanation
io.bytes.per.checksum,891744574,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.java.lang.NegativeArraySizeException: -564233426
	",property-item-hit	io
dfs.datanode.fsdatasetasyncdisk.max.threads.per.volume,4,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.java.lang.NegativeArraySizeException: -564233426
	",value-hit
io.bytes.per.checksum,891744574,ERROR	Task: attempt_1701312551309_0321_r_000000_0 - exited : java.lang.NegativeArraySizeException: -564233426	,property-item-hit	io
dfs.namenode.startup.delay.block.deletion.sec,0,ERROR	Task: attempt_1701312551309_0321_r_000000_0 - exited : java.lang.NegativeArraySizeException: -564233426	,value-hit
dfs.datanode.fsdatasetasyncdisk.max.threads.per.volume,4,ERROR	Task: attempt_1701312551309_0321_r_000000_0 - exited : java.lang.NegativeArraySizeException: -564233426	,value-hit
io.bytes.per.checksum,891744574,INFO	Diagnostics report from attempt_1701312551309_0321_r_000000_0: Error: java.lang.NegativeArraySizeException: -564233426	,property-item-hit	io
dfs.namenode.startup.delay.block.deletion.sec,0,INFO	Diagnostics report from attempt_1701312551309_0321_r_000000_0: Error: java.lang.NegativeArraySizeException: -564233426	,value-hit
dfs.datanode.fsdatasetasyncdisk.max.threads.per.volume,4,INFO	Diagnostics report from attempt_1701312551309_0321_r_000000_0: Error: java.lang.NegativeArraySizeException: -564233426	,value-hit
io.bytes.per.checksum,891744574,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	io
io.bytes.per.checksum,891744574,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	io
io.bytes.per.checksum,891744574,"WARN	Graceful stop failed. Exiting..java.lang.NullPointerException
	",property-item-hit	io
io.bytes.per.checksum,891744574,INFO	Exiting with status 1: java.lang.NullPointerException,property-item-hit	io
